src/githubHelper.ts

```diff
@@ -160,6 +160,62 @@ export default class GithubHelper {
 
   // ----------------------------------------------------------------------------
 
+  
+  /**
+   * Gets a release by tag name
+   * @param tag {string} - the tag name to search a release for
+   * @returns 
+   */
+   async getReleaseByTag(tag) {
+    try {
+      await utils.sleep(this.delayInMs);
+      // get an existing release by tag name in github
+      let result = await this.githubApi.repos.getReleaseByTag({
+        owner: this.githubOwner,
+        repo: this.githubRepo,
+        tag: tag
+      });
+
+      return result;
+    } catch (err) {
+      console.error('No existing release for this tag on github');
+      return null;
+    }
+  }
+
+  // ----------------------------------------------------------------------------
+
+  /**
+   * Creates a new release on github
+   * @param tag_name {string} - the tag name
+   * @param name {string} - title of the release
+   * @param body {string} - description for the release
+   */
+   async createRelease(
+    tag_name: string, 
+    name: string,
+    body: string) {
+    try {
+      await utils.sleep(this.delayInMs);
```

Comment:
Is the sleep here neccessary?

Reply:
same as above

https://github.com/piceaTech/node-gitlab-2-github/pull/94#discussion_r763250387